### PR TITLE
Show 'Or' between Age and DOB fields in Patient entry and edit

### PR DESCRIPTION
**Pull request by msasikanth**
https://app.clubhouse.io/simpledotorg/story/2932/show-or-between-age-and-dob-fields-in-patient-entry-and-edit

### CODE REVIEW

File: app/src/main/res/layout/screen_manual_patient_entry.xml [hunk 1]
```diff
@@ -55,7 +55,8 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_marginTop="@dimen/spacing_16"
-        android:paddingStart="@dimen/spacing_4">
```
Comment (janhavisinghh) [inline app/src/main/res/layout/screen_manual_patient_entry.xml:58]: I see that the separator was already there. What exactly was the reason for the separator to not be visible?

**Review by janhavisinghh — APPROVED** [review]
LGTM

File: app/src/main/res/layout/screen_manual_patient_entry.xml (hunk 1 above)
Comment (msasikanth) [reply app/src/main/res/layout/screen_manual_patient_entry.xml:58]: It's because of the margin top and gravity of the container.

**Pull request merged**